### ISSUE TITLE
Cell output is sometimes invisible until toggled

### ISSUE THREAD

**Issue by ckald** [ipython/ipython#7624]
From time to time after few recalculations of a cell (and possibly remote connection restorations), it's output does not appear. Cell identifier goes through regular steps `In [ ] -> In [*] -> In[4]` but the output is invisible. The workaround that works for me is to use `Cell -> Current Output -> Toggle`. After that `.btn.output_collapsed` appears and I can expand the cell output.

This is not critical, but is completely mysterious for non-advanced users of my public notebook server.

```bash
{'commit_hash': '4693727',
 'commit_source': 'repository',
 'default_encoding': 'UTF-8',
 'ipython_path': '/var/www/jupyter/ipython/IPython',
 'ipython_version': '3.0.0-dev',
 'os_name': 'posix',
 'platform': 'Linux-3.13.0-32-generic-i686-with-Ubuntu-14.04-trusty',
 'sys_executable': '/var/www/jupyter/env/bin/python',
 'sys_platform': 'linux',
 'sys_version': '3.4.0 (default, Apr 11 2014, 13:05:18) \n[GCC 4.8.2]'}
```

**Comment by dsblank** [comment]
Are you updating often and running from ipython-master? If so, I will see this behavior when the CSS changes. All is fine once the cache has been cleared.

**Comment by ckald** [comment]
Currently I use (this is the first and only version on my instance)
```bash
# pip freeze|grep ipython
-e git+https://github.com/ipython/ipython.git@469372778e4835a577fcea54f609ed4a39496c98#egg=ipython-master
```

I will try to clear the cache and will report again later

**Comment by minrk** [comment]
Should be fixed by #7631
